airflow/config_templates/config.yml

```diff
@@ -2680,7 +2680,14 @@ dag_processor:
                 "classpath": "airflow.dag_processing.bundles.git.GitDagBundle",
                 "kwargs": {
                     "subdir": "dags",
-                    "repo_url": "git@github.com:example.com/my-dags.git",
+                    "tracking_ref": "main",
+                    "refresh_interval": 0
+            },
+            {
+                "name": "my-git-repo",
+                "classpath": "airflow.dag_processing.bundles.git.GitDagBundle",
+                "kwargs": {
+                    "subdir": "dags",
                     "tracking_ref": "main",
                     "refresh_interval": 0
```

Comment:
```suggestion
```

We don't really need 2 examples of the same bundle config in there :)

Reply:
Thanks, let me only left 1 example in bundle config.